bitnami/harbor/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   category: Infrastructure
 apiVersion: v2
-appVersion: 2.4.0
+appVersion: 2.5.0
```

Comment:
Please do not change the `appVersion` field of the `Chart.yaml`. Only modifying the field `version` is required.